packages/connections/src/modules/telemetry.ts

```diff
@@ -117,6 +120,7 @@ export function trackNewConnectionEvent(
         server_version: build.version,
         server_arch: host.arch,
         server_os_family: host.os_family,
+        topology_type: dataService.currentTopologyType(),
```

Comment:
@mmarcon Can you verify that this is okay?

`topology_type: Single | ReplicaSetNoPrimary | ReplicaSetWithPrimary | Sharded | LoadBalanced | Unknown`

In the context of Socks5, this is helpful because it lets us verify that users can use Socks5 or SSH to connect to replica sets, but I figured it might be nicer to expose this the way we get it from the driver rather than using a boolean replicaset/not-replicaset flag?

Reply:
As long as `Single | ReplicaSetNoPrimary | ReplicaSetWithPrimary | Sharded | LoadBalanced | Unknown` are all strings, that looks good to me. We can easily filter and group by them in Amplitude.